### DISCUSSION TITLE
github won't verify my student status

### DISCUSSION THREAD

**Discussion by zxvum** [community/community#59845] [GitHub Education]
### Hi everyone,

Hello everyone, I am a student of grade 9, currently transferred to grade 10, I live in Russia, every time I send an application with a photo of paper confirmation of my studies at school N3, I immediately receive disapproval, paper confirmation is filled out manually by the director of my school, what should I do? I really really want to try copilot and all the functions of organizations🥲

**Comment by CoolGenius-123** [comment]
Same its happening immediately rejection and I am getting to get my college email our college don't provide any kind of college email what should I do ?

**Comment by samus-aran** [comment]
Hi @zxvum  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

**Reply by CoolGenius-123** [reply]
They are basically asking us to provide student email , but our college don't provide Student email, even when we are applying through official documents which I got through earlier its not working now .

**Reply by CoolGenius-123** [reply]
I know there are now-a-days lot's of  bad guys are there who are using bad means to get Student pack , but in this thing lot's of genuine and good people who really need it they are suffering in it , so please consider solving this 🥺.

**Comment by nonvenomous** [comment]
Hi @CoolGenius-123 I'm in the same boat.
Here a bigger thread https://github.com/orgs/community/discussions/59360
Consider upvoting and commenting.
